scripts/FIWARELabNodesSLA/README.org

```diff
@@ -0,0 +1,74 @@
+#+TITLE:     FIWARE Lab Nodes SLA
+#+AUTHOR:    Katarzyna Di Meo
+#+DESCRIPTION: Python app for nodes' SLA computing
+#+KEYWORDS:  python, sla, nodes, fiware
+#+LANGUAGE:  en
+
+#+html: <p align="left"><a href="http://www.apache.org/licenses/LICENSE-2.0"><img src="https://camo.githubusercontent.com/651d047a62d409c28b23517069c652af8b8c429c/68747470733a2f2f636f636f61706f642d6261646765732e6865726f6b756170702e636f6d2f6c2f526573744b69742f62616467652e706e67" /></a></p>
+
+* General description
+  The nodes, that are funded by the FI-NEXT project, have to assure a certain level of quality defined through a Service Level Agreement (SLA). One of the conditions that must be satisfied is the services availability on the node that must be above the 95% threshold.\\
```

Comment:
The nodes taking part to FIWARE Lab have to assure a certain level of quality defined through a Service Level Agreement (SLA). One of the conditions that must be satisfied is the services availability on the node that must be above the 95% threshold.\\